.github/workflows/sphinx-check-page-multiversion.yml

```diff
@@ -1,7 +1,22 @@
-name: "Build Multiversion Page (PR)"
+name: "Build Multiversion Page"
 on:
   workflow_dispatch:
+    inputs:
+      BASE_BRANCH:
+        description: 'Chose branch to run on'
+        required: true
+        default: 'master'
+        type: choice
+        options:
+        - humble
+        - iron
```

Comment:
@christophfroehlich should we add jazzy already as a branch option as we know we are going to have it soon?

Reply:
there will be more changes necessary, I'll do this all at once